packages/components/src/styles/components/app-frame.scss

```diff
@@ -12,8 +12,9 @@
   grid-template-areas:
     "header header"
     "sidebar main"
-    "sidebar footer";
-  grid-template-rows: auto 1fr auto;
+    "sidebar footer"
+    "sidebar sticky-footer";
```

Comment:
> If there's a possibility that this grid will eventually have gap properties

@philrenaud I don't see this as a possibility. If spacing will be added, (**if**), it will be in the "main" content area
- or in the form of internal padding (I doubt it, but may as well be if we decide to have standard, responsive padding for all the consumers, at page level) 
- or delegated to a dedicated component (imagine like a page-level "main" layout, similar to what it's done in cloud UI with the `<Core::Layout::Main>` component.